project_task_materials_analytic_partner/__openerp__.py

```diff
@@ -19,6 +19,4 @@
         'views/project_view.xml',
     ],
     'installable': True,
-    'autoinstall': True,
```

Comment:
I'm OK with it, just checking if it wasn't a mistake.
It would be nice to note this in the README.